docs/user-guide/index.md

```diff
@@ -7,4 +7,5 @@ maxdepth: 1
 ---
 installation
 histogram_mode_detectors
```

Comment:
Strange, this seems to not appear in the built docs:
![Screenshot_20241127_102218](https://github.com/user-attachments/assets/2723c747-8a43-4b08-8ba4-1ebb9140a71a)